docs/static/spec/rasa.yml

```diff
@@ -1196,7 +1196,7 @@ components:
           $ref: '#/components/schemas/ParseResult'
       required: ["text", "sender"]
 
-    Entity:
+   Entity:
```

Comment:
This over-indentation is causing a parsing error [here](https://github.com/RasaHQ/rasa/actions/runs/4058149748/jobs/7001210941#step:13:22), please revert.